### DISCUSSION TITLE
Adding Arabic (ar-*) language ?

### DISCUSSION THREAD

**Discussion by Bodhisatvva** [deepgram/community#787] [General help] [answered]
I would like to know about your plans to add Arabic language specifically (egyptian arabic "arz") support to the system, Could you please provide an estimated date or any updates regarding this feature

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment, marked as answer]
Arabic isn't currently on our roadmap. Right-to-left languages raise some unique challenges, and we're prioritizing other new products and features over that set of languages. However, you could try our hosted whisper model: `language=ar&model=whisper`.

**Comment by liferays** [comment]
any plans in the near future for Arabic support?

**Reply by jpvajda** [reply]
@liferays 👋  I don't have any specific timelines on the support of Arabic at this time.
